README.md

```diff
@@ -1,3 +1,101 @@
+* Run serverless, stateful containers that enable pay-for-use billing
+
+### A note on microservices
+This project is organized into microservices. Although this repository contains a few top level files such as this README.md, each direct subfolder of this project is its own independent microservice. This means a few things: 
+* Each microservice subfolder is meant to be built and deployed independently from each other
+* The microservices interact with each other in deployment through HTTP requests
+    * You will need to supply environmental variable to the microservices specifying each others' endpoints
+* Each subfolder has its own README.md with more specific information regarding building, testing, deployment, etc.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system. Because of this microservice architecture, each subfolder will need to be deployed as its own independent subfolder. For specific instructions, see the README.md file in each subfolder, but generally, they all follow the following pattern:
+
+### Prerequisites
+
+These microservices are written in [Python 3](https://www.python.org/) and should run on Python 3.6+.
+
+### Installing
+
+How to install and run each microservice for local development.
+
+Make sure your working directory is the microservice's subfolder. For example, for the "users" microservice:
+
+```
+cd users
+```
+
+Set up and activate a virtual environment.
+
+```
+python3 -m venv venv && . venv/bin/activate
+```
+
+Install the required python modules.
+
+```
+pip3 install requirements.txt
+```
+
+### Running the service
+
+Use flask to run the microservice locally.
+
+```
+FLASK_APP=app.py && flask run
+```
+
+## Running the tests
+
+Testing again is handled on the microservice level. To test, ensure your working directory is the microservice's subfolder then use `unittest`.
+
+```
+python3 -m unittest test.py
+```
+
+## Deployment
+
+This project is set up for continuous deployment to Google Cloud Run (managed) via [Cloud Build](https://cloud.google.com/run/docs/continuous-deployment), and it will automatically be built and deployed on pushes to the master branch of [this repository](https://github.com/knative-portability/large-events).
+
+You can also [manually deploy this project to a service like Cloud Run](https://cloud.google.com/run/docs/quickstarts/build-and-deploy). For example, to deploy one of the microservices to the service [SERVICE] in the GCP project with ID [PROJECT-ID] run:
+
+```
+gcloud builds submit --tag gcr.io/[PROJECT-ID]/[SERVICE]
+gcloud beta run deploy --image gcr.io/[PROJECT-ID]/[SERVICE] --platform managed
+```
+
+## Built With
+
+* [Flask](http://flask.pocoo.org/) - Python web service framework
+* [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) - Cloud NoSQL database
```

Comment:
If MongoDB needs to be provisioned, you might want to mention it in the "Deployment" section.

Reply:
Done https://github.com/knative-portability/large-events/pull/6/commits/1234525d514f077d41dae372ca64598e2beb2c0e